nixos/modules/services/misc/gitlab.xml

```diff
@@ -97,6 +97,12 @@ services.gitlab = {
    state folder.
   </para>
 
+  <para>
+    When <literal>icoming_mail.enabled</literal> is set to <literal>true</literal>
+    in <link linkend="opt-services.gitlab.extraConfig">extraConfig</link> an additional
+    service called <literal>gitlab-mailroom</literal> is enabled for fetching incoming mail.
```

Comment:
How do I configure this service? Does it use whatever sendmail is configured? Is there a configuration example somewhere we could link to?

Reply:
I don't think we want to [copy the upstream manual](https://docs.gitlab.com/ee/administration/incoming_email.html) here and I don't feel like their links are stable at all (like their repo layouts). Also, sendmail has nothing to do with it, since this is about receiving mail and not sending it ;)

Reply:
Right - `incoming_mail.enabled` is probably enough of breadcrumbs for people to scroll down to "Example for source installs:" in https://docs.gitlab.com/ee/administration/incoming_email.html.